HelpSource/Guides/standalones.schelp

```diff
@@ -3,11 +3,11 @@ SUMMARY:: A guide to creating SuperCollider-based applications
 CATEGORIES:: Platform>OSX, Tutorials
```

Comment:
@vivid-synth would it be as simple as replacing all `Platform>OSX` with `Platform>macOS`?